docs/search/pg-trgm-performance.md

```
@@ -0,0 +1,144 @@
+## 6. 기존 LIKE 검색 실행계획
+
+| 항목 | 결과 |
+| --- | --- |
+| 주요 Scan | `Parallel Seq Scan on documents` |
+| Filter | `title ~~ '%사업계획서%' AND member_id = 1` |
+| Rows Removed by Filter | `99400` per worker |
+| Sort | `top-N heapsort` |
+| Buffer hit | `1914` |
+| Execution Time | `10.003ms` |
+
+```mermaid
```

markdownlint-cli2 (0.22.1)

[warning] Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)